src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -308,11 +337,32 @@ public static class StandardTrainersCatalog
         {
             Contracts.CheckValue(catalog, nameof(catalog));
             var env = CatalogUtils.GetEnvironment(catalog);
-            return new SdcaMulticlassTrainer(env, labelColumnName, featureColumnName, exampleWeightColumnName, loss, l2Regularization, l1Threshold, maximumNumberOfIterations);
+            return new SdcaNonCalibratedMulticlassTrainer(env, labelColumnName, featureColumnName, exampleWeightColumnName, loss, l2Regularization, l1Threshold, maximumNumberOfIterations);
+        }
+
+        /// <summary>
+        /// Predict a target using a maximum entropy classification model trained with <see cref="SdcaCalibratedMulticlassTrainer"/> and advanced options.
+        /// </summary>
+        /// <param name="catalog">The multiclass classification catalog trainer object.</param>
+        /// <param name="options">Trainer options.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[SDCA](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/StochasticDualCoordinateAscentWithOptions.cs)]
+        /// ]]></format>
+        /// </example>
+        public static SdcaCalibratedMulticlassTrainer SdcaCalibrated(this MulticlassClassificationCatalog.MulticlassClassificationTrainers catalog,
```

Comment:
>SdcaCalibratedMulticlassTrainer [](start = 22, length = 31)

nit: if you are doing another iteration, would be great if you keep the two extensions for `SdcaCalibratedMulticlassTraniner` one after the other.

Reply:
Will fix it in next iteration.

---
In reply to: [267060562](https://github.com/dotnet/machinelearning/pull/2976#discussion_r267060562) [](ancestors = 267060562)